frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemDestination/uploadListItemDestination.styles.ts

```diff
@@ -16,20 +16,44 @@
  */
 
 import styled from 'styled-components';
-import { Typography } from '@controls/typography';
 
-export const Container = styled.div`
-	background-color: ${({ theme }) => theme.palette.tertiary.lightest};
-	color: ${({ theme }) => theme.palette.secondary.main};
-	border-radius: 5px;
-	padding: 9px 12px;
-	width: 271px;
-	margin: 0 10px;
+import { TextField } from '@material-ui/core';
+import { ErrorTooltip } from '@controls/errorTooltip';
 
-	box-sizing: border-box;
+export const TextInput = styled(TextField)`
+	margin: 0;
+	width: 271px;
+	border: none;
+	>.MuiInputBase-root {
+		>.MuiInputBase-input {
+			font-weight: bold;
+		}
+		${({ state, theme, error }) => {
+		if (error) return '';
```

Comment:
I dont think is necessary to handle 'error'. It already returns '' if its not 'new' or 'existing'

Reply:
The problem here was that an input can have a state and an error at the same time (the state just says whether it's a new container, existing container, or is empty) and the state overwrites the error style. I suppose the term 'state' is misleading and I should instead use something else

Reply:
I understand that, but I tried it and I didnt noticed anything different between one version taking error as a parameter and the other without it.